src/libs/PersonalDetailsUtils.ts

```diff
@@ -224,14 +225,25 @@ function getStreetLines(street = '') {
     return [streets[0], streets[1]];
 }
 
+/**
+ * Get the current address from addresses array
+ *
+ * @param privatePersonalDetails - details object
+ * @returns - current address object
+ */
+function getAddress(privatePersonalDetails: OnyxEntry<PrivatePersonalDetails>): Address | undefined {
+    const {addresses} = privatePersonalDetails ?? {};
+    return addresses?.[addresses.length - 1];
```

Comment:
Slight change needed here. There's an optional `current` boolean on each address object. Can we return the address with that set instead? If there is none found, we can fall back to `addresses?.[addresses.length - 1];` 

```suggestion
    const currentAddress = addresses?.find((address) => address.current);
    return currentAddress ?? addresses?.[addresses.length - 1];
```